.github/workflows/tests.yml

```diff
@@ -15,19 +15,25 @@ jobs:
     runs-on: macOS-latest
     strategy:
       matrix:
-        flags: [
-          '',
-          '--use-libraries',
-          '--use-static-frameworks'
+        podspec: [GoogleSignIn.podspec, GoogleSignInSwift.podspec]
+        flag: [
+          "", 
+          "--use-libraries", 
+          "--use-static-frameworks"
         ]
+        include:
+          - podspec: GoogleSignInSwift.podspec
+            includePodspecFlag: "--include-podspecs='*.podspec'"
```

Comment:
Now that we're only targeting the Swift pod run, we could be more specific:

`--include-podspecs='GoogleSignIn.podspec'`